lib/std/child_process.zig

```diff
@@ -749,38 +748,38 @@ fn windowsCreateProcess(app_name: [*:0]u16, cmd_line: [*:0]u16, envp_ptr: ?[*]u1
 
 /// Caller must dealloc.
 fn windowsCreateCommandLine(allocator: *mem.Allocator, argv: []const []const u8) ![:0]u8 {
-    var buf = try ArrayListSentineled(u8, 0).initSize(allocator, 0);
+    var buf = try ArrayList(u8).init(allocator);
     defer buf.deinit();
-    const buf_stream = buf.outStream();
+    const buf_wi = buf.outStream();
```

Comment:
If we're doing cleanup, I guess this should use `writer` since `outStream` is getting deprecated?

```suggestion
    const buf_wi = buf.writer();
```

Reply:
On second thought, is `buf_wi` even needed here at all?

Reply:
I removed all uses of `buf_stream` as they were unceseccary, but then forgot to remove this. Will update.